core/src/utils/test/aria.spec.ts

```diff
@@ -5,7 +5,8 @@ import { Label } from '../../components/label/label';
 import { Toggle } from '../../components/toggle/toggle';
 import { getAriaLabel } from '../helpers';
 
-describe('getAriaLabel()', () => {
+// TODO FW-5969
+describe.skip('getAriaLabel()', () => {
```

Comment:
These tests use toggle which are failing now because we removed support for the legacy form syntax. I'm not 100% sure if this utility is still needed, so I made a tech debt ticket to investigate after all of the form controls have been migrate.